modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/flow/execution/v1/FlowExecutionPositiveTest.java

```diff
@@ -113,6 +126,15 @@ public void executeFlow() throws Exception {
         Assert.assertNotNull(response.getData());
     }
 
+    @Test(dependsOnMethods = "executeFlow")
+    public void verifyUserLogin() throws Exception {
+
+        String userId = UserUtil.getUserId(USER, context.getContextTenant());
+        unlockUser(userId);
+        JSONObject authenticationResponse =  authenticatorRestClient.login(USER, "Wso2@test");
```

Comment:
At the end of the test execution let's close the REST client and lock the user as initially intended.